kuka_rox_hardware_interface/include/kuka_rox_hw_interface/robot_manager_node.hpp

```diff
@@ -64,6 +66,15 @@ class RobotManagerNode : public kroshu_ros2_core::ROS2BaseLCNode
   std::shared_ptr<rclcpp_lifecycle::LifecyclePublisher<std_msgs::msg::Bool>> is_configured_pub_;
   std_msgs::msg::Bool is_configured_msg_;
 
+  const std::string POSITION_CONTROL = "position";
+  const std::string TORQUE_CONTROL = "torque";
+  const std::string IMPEDANCE_CONTROL = "impedance";
+
+  const std::string POSITION_CONTROLLER_NAME = "position_controller_name";
+  const std::string IMPEDANCE_CONTROLLER_NAME = "impedance_controller_name";
+  const std::string TORQUE_CONTROLLER_NAME = "torque_controller_name";
```

Comment:
shouldn't this be called TORQUE_CONTROLLER_NAME_PARAM? As I see, it isn't the name itself, but the name of the parameter which defines the controller's name.

Reply:
good suggestion, thanks